JS-Algorithms/challenges.js

```diff
@@ -118,7 +173,20 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  // offset index due to 1,1 being first two elements of fibonacci sequence
+  let index = 3 
+  let current = 1
+  let next = 1
+
+  while (true) {
+    let fib = current + next
+    if(fib.toString().length >= n) return index
```

Comment:
Nit: even if the conditional only has one inner statement, we try to make that really explicit by adding adding brackets.

Also note how I added the semi-colon `;` to the end of the line

```suggestion
    if(fib.toString().length >= n) {
      return index;
    }
```